src/libraries/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- allow use of C# 9 'record' type -->
+    <LangVersion>preview</LangVersion>
```

Comment:
LangVersion is already set to preview:

https://github.com/dotnet/runtime/blob/74b173787bad85eac022b9fb680f247aeda76305/Directory.Build.props#L92

Does it work without adding this to the test project?

Reply:
It should; changing.